app/stylesheet/menu.scss

```diff
@@ -118,6 +118,10 @@
         border-color: transparent $inverse-01 transparent transparent !important;
       }
 
+      .pficon {
+        color: $field-01;
```

Comment:
What's this doing?

Reply:
After doing bin/update, there is one more menu item added in my lefnav menu, it is a pficon , not an svg, it is in blue color instead of above color. so i am overriding it so that that icon seems consistent with other svgs.